.idea/.name

```diff
@@ -0,0 +1 @@
+Tutorial
```

Comment:
Do you think it's correct to have all these idea/studio files in here? I thought it would be cleaner just to have a gradle build files.